components/dialog/dialog.js

```diff
@@ -252,19 +258,21 @@ export class Dialog extends HTMLElement {
 
         let bodyElement = this.querySelector("[slot=body]");
 
-        if(bodyElement == null) {
+        if (bodyElement == null) {
             bodyElement = document.createElement("div");
             bodyElement.setAttribute("slot", "body");
             this.appendChild(bodyElement);
+        } else {
+            bodyElement.textContent = "";
```

Comment:
Let's maybe chat on this. I want to understand why we need to set the textContent on the bodyElement to empty string if the bodyElement is already defined

Reply:
replaced .textContent with .replaceChildren to prevent parent dialog data appending to child dialog data.